opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -115,19 +124,7 @@ namespace Opm {
         }
     }
 
-    void Schedule::createTimeMap(const Deck& deck) {
-        boost::gregorian::date defaultStartTime( 1983, 1, 1 );
-        boost::posix_time::ptime startTime( defaultStartTime );
-
-        if (deck.hasKeyword("START")) {
-             const auto& startKeyword = deck.getKeyword("START");
-            startTime = TimeMap::timeFromEclipse(startKeyword.getRecord(0));
-        }
-
-        m_timeMap.reset(new TimeMap(startTime));
-    }
-
-    void Schedule::iterateScheduleSection(const ParseContext& parseContext , const SCHEDULESection& section, IOConfigPtr ioConfig) {
+    void Schedule::iterateScheduleSection(const ParseContext& parseContext , const SCHEDULESection& section, IOConfigPtr ) {
```

Comment:
Please remove `IOConfigPtr` here, and then remove `ioConfig` variable in constructor parameter list.